test/index.js

```diff
@@ -425,4 +425,19 @@ describe("Tallahassee", () => {
     });
 
   });
+
+  describe("options", () => {
+    describe("htmlparser2 options", () => {
+      it("passes on options to htmlparser2", async () => {
+        const markup = "<svg><text>åäö</text></svg>";
+        const htmlparser2 = {
+          xmlMode: true,
+          decodeEntities: true
+        };
+        const browser = await new Browser({ htmlparser2 }).load(markup);
+        expect(browser.$.html()).to.not.include("<html>");
```

Comment:
What is the purpose of this? Tallahassee should mimic a browser, so I would expect it to wrap its content in html.

Reply:
the assertion is just to make sure that options are passed along.
the actual purpose of enabling this is to verify that no illegal (ESI) dollar signs are output in solveig. with `encodeEntities: false` we can't see a difference between `&dollar;` and an actual `$`. it's actually the only option we need but figured it'd be less complex just passing along entire options object.